pennylane/estimator/ops/qubit/non_parametric_ops.py

```diff
@@ -574,7 +713,7 @@ def pow_resource_decomp(
         """
 
         if (mod_8 := pow_z % 8) == 0:
-            return [GateCount(resource_rep(Identity))]
+            return [GateCount(resource_rep(qre.Identity))]
```

Comment:
Why is this getting changed int he adjoint and controlled PR?

Reply:
I had to change the way I import operators to deal with circular import errors which starting emerging when symbolic resource operators were added.